src/trixi_include.jl

```diff
@@ -0,0 +1,145 @@
+# Note: We can't call the method below `Trixi.include` since that is created automatically
+# inside `module Trixi` to `include` source files and evaluate them within the global scope
+# of `Trixi`. However, users will want to evaluate in the global scope of `Main` or something
+# similar to manage dependencies on their own.
```

Comment:
```suggestion
# Note: We can't call the method below `TrixiBase.include` since that is created automatically
# inside `module TrixiBase` to `include` source files and evaluate them within the global scope
# of `TrixiBase`. However, users will want to evaluate in the global scope of `Main` or something
# similar to manage dependencies on their own.
```